src/github.com/matrix-org/dendrite/clientapi/routing/memberships.go

```diff
@@ -30,6 +30,10 @@ type response struct {
 	Chunk []gomatrixserverlib.ClientEvent `json:"chunk"`
 }
 
+type joinedResponse struct {
```

Comment:
This would better be inside gomatrix if it isn't there already I think

Reply:
TBH, that's probably more faff than its worth